src/foundry/common/abstract/document.d.mts

```diff
@@ -871,7 +871,7 @@ declare abstract class Document<
 
   /**
    * @deprecated since v12, will be removed in v14
-   * @remarks `"The Document._onCreateDocuments static method is deprecated in favor of Document._onCreateOperation"`
+   * @remarks "The `Document._onCreateDocuments` static method is deprecated in favor of {@link Document._onCreateOperation | `Document._onCreateOperation`}"
```

Comment:
Nit; you can use `@linkcode`. I suppose you may be avoiding that due to not having the TSDoc change in this PR?